crates/pallet-domains/src/domain_registry.rs

```diff
@@ -334,19 +334,8 @@ pub(crate) fn calculate_max_bundle_weight_and_size(
                 .checked_mul(consensus_slot_probability.0)?,
         )?;
 
-    // This represents: Ceil[2*Sqrt[bundle_slot_probability/SLOT_PROBABILITY]])
-    let std_of_expected_bundles_per_block = expected_bundles_per_block
```

Comment:
maybe i'm missing something here but shouldn't we be able to update the `Ceil[2*Sqrt[bundle_slot_probability/SLOT_PROBABILITY]])` to achieve the same or something closer rather than removing this completely ?
Is this temporary or we decided to remove this completely ? @dariolina

Reply:
We can remowe it for now. I advocate for putting the bundle limit into domain config rather than doing this calculation (FYI https://github.com/subspace/protocol-specs/pull/22/files)